.env.example

```diff
@@ -4,9 +4,9 @@
 # env variables required for running the demo
 
 # mandatory: URL of the container registry to use.
-# Default (local Kind registry): localhost:5000
+# Default (local Kind registry): 127.0.0.1:5000
 # Azure Container Registry (ACR) example: osmci.azurecr.io/osm
-export CTR_REGISTRY=localhost:5000
+export CTR_REGISTRY=127.0.0.1:5000
```

Comment:
We could also do `0.0.0.0` @nojnhuh  - your call. :)

Reply:
I'm fine to keep it as 127.0.0.1